.gitignore

```diff
@@ -4,6 +4,7 @@
 build/
 dist/
 *.egg-info/
+.vscode/
```

Comment:
Please do not add editor/IDE specific files to the gitignore. You can also create a user specific gitignore file and add it to core.excludefile (seehttps://stackoverflow.com/questions/7335420/global-git-ignore).

Reply:
uhh great, thanks for that tip 🙂 👍